### PR TITLE
Fixing and finishing unit tests

### PR DESCRIPTION
**Pull request by mestizoLopez**
# Description

Unit Tests For DaprHttp

## Issue reference

Please reference the issue this PR will close: #_[21]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ X ] Code compiles correctly
* [ X ] Created/updated tests
* [ X ] Extended the documentation

### CODE REVIEW

File: sdk/src/test/java/io/dapr/client/DaprHttpTest.java [hunk 1]
```diff
@@ -74,7 +94,7 @@ public void invokeGetMethod() throws IOException {
                 .get("http://localhost:3500/v1.0/get")
                 .respond(EXPECTED_RESULT);
 
-        DaprHttp daprHttp = new DaprHttp(3500, okHttpClient);
+        DaprHttp daprHttp = new DaprHttp(3500,okHttpClient);
```
Comment (artursouza) [inline sdk/src/test/java/io/dapr/client/DaprHttpTest.java]: This thing will be resolved in our checkstyle. Usually, I see checkstyles failing without this space. It is OK now, but we will do a big reformatting in the end of the deliverable.

**Comment by artursouza** [comment]
> # Description
> _Please explain the changes you've made_
> 
> ## Issue reference
> We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.
> 
> Please reference the issue this PR will close: #_[issue number]_
> 
> ## Checklist
> Please make sure you've completed the relevant tasks for this PR, out of the following list:
> 
> * [ ]  Code compiles correctly
> * [ ]  Created/updated tests
> * [ ]  Extended the documentation

Please, fill in the description for this PR.

File: sdk/src/test/java/io/dapr/client/DaprHttpTest.java [hunk 2]
```diff
@@ -91,35 +111,79 @@ public void invokeMethodWithHeaders() throws IOException {
         headers.put("header1","value1");
 
         mockInterceptor.addRule()
-                .get("http://localhost:3500/v1.0/get?header1=value1&header=value")
```
Comment (artursouza) [inline sdk/src/test/java/io/dapr/client/DaprHttpTest.java]: Do we have any other test for params?

File: sdk/src/test/java/io/dapr/client/DaprHttpTest.java (hunk 1 above)
Comment (mestizoLopez) [reply sdk/src/test/java/io/dapr/client/DaprHttpTest.java]: Sorry i missed that detail. i am going to check and fix if there is any other styling problem.

File: sdk/src/test/java/io/dapr/client/DaprHttpTest.java (hunk 2 above)
Comment (mestizoLopez) [reply sdk/src/test/java/io/dapr/client/DaprHttpTest.java]: I added another one for parameters. When i fix the styling issue, i will push that test as well.

File: sdk/src/test/java/io/dapr/client/DaprHttpTest.java (hunk 1 above)
Comment (artursouza) [reply sdk/src/test/java/io/dapr/client/DaprHttpTest.java]: We will do a big restyling change once code is stable.

**Comment by artursouza** [comment]
Please, update the description of the PR.

**Comment by mestizoLopez** [comment]
Done. Sorry to Not update the description before

**Pull request merged**